drf_fsm_transitions/viewset_mixins.py

```diff
@@ -22,7 +22,7 @@ def inner_func(self, request, pk=None, **kwargs):
     return inner_func
 
 
-def get_viewset_transition_action_mixin(model, **kwargs):
+def get_viewset_transition_action_mixin(model, field='status', **kwargs):
```

Comment:
field='status' should be field='state' (default value is state)

Reply:
updated